internal/decision/engine.go

```diff
@@ -22,6 +22,27 @@ import (
 	peer "github.com/libp2p/go-libp2p-core/peer"
 )
 
+// Log only for a specific peer we want to follow.
+func LogTargetPeer(p peer.ID, template string, args ...interface{}) {
+	//targetPeerString := os.Getenv("BITSWAP_LOG_PEER") // FIXME: Initialize and parse once.
+	//if targetPeerString == "" {
+	//	return
+	//}
+	// FIXME: Hardcoded from util/vole.go.
+	targetPeerString := "12D3KooWDpJ7As7BWAwRMfu1VU2WCqNjvq387JEYKDBj4kx6nXTN"
+
+	targetPeer, err := peer.Decode(targetPeerString)
+	if err != nil {
+		panic("invalid log target peer " + err.Error())
+	}
```

Comment:
Even though this is debugging code, I'd initialize this statically (in an `init` function) to avoid a _lot_ of allocations and string parsing.

Reply:
Yes, especially since we hardcode the peer address we track so this isn't going to change on the fly.